packages/builder/src/components/design/PropertiesPanel/PropertyControls/OptionsEditor/OptionsEditor.svelte

```diff
@@ -7,22 +7,55 @@
 
   export let value
 
+  let drawerVisible = false
```

Comment:
Rather than keeping track of this and resetting the validation whent he drawer is hidden, it would be better to instead just wipe valiation when showing the drawer. We could update the handler for clicking on the "define options" button to both reset the validation and then show the drawer. That's a bit cleaner and means we don't need to change the drawer itself.